.github/workflows/test.yml

```
@@ -0,0 +1,38 @@
+---
+name: Tests
+
+on: [push, pull_request]
+
+jobs:
+  tests:
+    if: (github.event_name == 'push') ||
+      (github.event_name == 'pull_request' &&
+      github.event.pull_request.head.repo.full_name != github.repository)
+
+    runs-on: ubuntu-22.04
+
+    steps:
+      - uses: actions/checkout@master
+
+      - name: Setup Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: '1.19'
+
```

A latest stable versions:
```suggestion
      - uses: actions/checkout@v3

      - name: Setup Go
        uses: actions/setup-go@v4
        with:
          go-version: '1.19'

```

---

Changed.